src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/SearchApprenticeshipVacanciesRequestValidator.cs

```diff
@@ -13,6 +13,15 @@ public SearchApprenticeshipVacanciesRequestValidator()
             RuleForEach(r => r.StandardCodes)
                 .Must(BeValidNumber)
                 .WithMessage((c, t) => $"{t} is invalid, expected a number.");
+
+            RuleFor(r => r.PageSize)
+                .GreaterThanOrEqualTo(1)
+                .WithMessage("Page size should be greater than or equal to 1")
+                .LessThanOrEqualTo(250)
```

Comment:
Could you stick this 250 in a constant, MaximumPageSize

Reply:
Fixed.

Reply:
General comment: Should we move all messages into resx or similar? This will help ensure we keep to a consistent approach with regards human readable communication. It also makes the life of a Content Designer easier as we can give them a single file to review all messages across the system.

Also I know there is no requirement for multiple languages but if we abstract the getting of the messages a little now it will make handling multiple languages, a/b testing, etc much easier in the future

Reply:
Valid point @chrisjensenuk. There is a separate task already recorded to generate Error Codes similar to SOAP Api, perhaps we could include this with it.

Reply:
I have removed the messages as Fluent Validation generates a message. We can use this for now as it will be standard across.